tests/unit/ViolationUtilsTest.ts

```diff
@@ -62,12 +63,13 @@ describe('getViolationsOnyxData', () => {
     describe('policyRequiresCategories', () => {
         beforeEach(() => {
             policyRequiresCategories = true;
-            policyCategories = {Food: {enabled: true}};
+            // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
NAB: I'm guessing this is for 'GL Code', but we can omit that if it's not required to be set, and get rid of the suppressions here and below. I see 'GL Code' isn't optional in PolicyTag, but I think it _could_ be. Checking both old dot and onyx in new dot, I have tags that don't have GL Code set